charts/kcp/templates/promtail.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+data:
+  promtail.yaml: |
+    server:
+      disable: true
+    positions:
+      filename: {{ .Values.audit.log.dir }}/positions.yaml
+    scrape_configs:
+    - job_name: kcp-audit
+      static_configs:
+      - labels:
+          __path__: {{ .Values.audit.log.dir }}/kcp.log
+          kubernetes_container_name: kcp
+          kubernetes_pod_name: ${HOSTNAME}
+          kubernetes_namespace_name: {{ .Release.Namespace }}
+    clients:
+    - bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+      tls_config:
+        ca_file: /run/secrets/kubernetes.io/serviceaccount/service-ca.crt
+      url: https://logging-loki-gateway-http.openshift-logging.svc:8080/api/logs/v1/application/loki/api/v1/push
```

Comment:
Does this need to be configurable? Or maybe we don't include this for now?

Reply:
I did consider not including promtail and other optional stuff, but it will make keeping the the repos in sync harder so I just disabled audit in the `values.yaml` instead

If we want this chart to be more minimal I can go ahead and strip out promtail, parca and other optional pieces, but then I think we'll want to think about how to automate the sync vs just a copy of the templates

Reply:
promtail changes now removed